### DISCUSSION TITLE
Welcome to mrv2 Discussions!

### DISCUSSION THREAD

**Discussion by ggarra13** [ggarra13/mrv2#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eblioxfr** [comment]
Hello there, I really find this player very attractive and useful. But it lacks one feature that is really important to me, so do you really plan to implement webp support?

**Comment by ggarra13** [comment]
@eblioxfr I don't know if you are the same person that contacted me privately, but here's what I responded to him:

It depends.  If you need webp decoding it should, I believe, be trivial to add it to the next version.  If you need webp saving, that might be more tricky.  Finally, if you need animated webp sequences, I don't have an ETA.

**Comment by eblioxfr** [comment]
yes, the same one, thanks for the reply!

**Comment by ggarra13** [comment]
@eblioxfr mrv2 v1.4.9 and vmrv2 v1.4.9 adds support for .webp images and sequences.  Note that animated .webp pictures are not supported nor is saving in the format.  Both would require some significant effort to implement.  If you want to pursue that, contact me privately for a quote.
